builder/builder-next/adapters/containerimage/pull.go

```diff
@@ -524,7 +524,10 @@ func (p *puller) Snapshot(ctx context.Context, g session.Group) (cache.Immutable
 		handlers         []images.Handler
 	)
 	if p.desc.MediaType == images.MediaTypeDockerSchema1Manifest {
-		schema1Converter = schema1.NewConverter(p.is.ContentStore, fetcher)
+		schema1Converter, err = schema1.NewConverter(p.is.ContentStore, fetcher)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Looks like `stopProgress()` should be called here; not sure why we don't use a `defer` for that though (I see a `defer` is defined on line 569, but before that, it's all manual)

```
builder/builder-next/adapters/containerimage/pull.go:498:2: lostcancel: the stopProgress function is not used on all paths (possible context leak) (govet)
	pctx, stopProgress := context.WithCancel(ctx)
	^
builder/builder-next/adapters/containerimage/pull.go:529:4: lostcancel: this return statement may be reached without using the stopProgress var defined on line 498 (govet)
			return nil, err
			^
```